rundeckapp/grails-app/controllers/rundeckapp/UrlMappings.groovy

```diff
@@ -51,6 +51,7 @@ class UrlMappings {
         "/api/$api_version/job/$id"(controller: 'scheduledExecution') {
             action = [GET: 'apiJobExport', DELETE: 'apiJobDelete']
         }
+        "/api/$api_version/job/$id/components"(controller: 'scheduledExecution', action: 'apiJobDefinitionComponentsValues')
```

Comment:
We will need another endpoint too: e.g. `/api/$api_version/jobs/components`, that includes the component metadata without component values, so that we can render the input properties in e.g. the New Job form (no existing job).